assembl/static2/js/app/components/administration/landingPage/manageModules.jsx

```diff
@@ -57,6 +85,10 @@ export const DumbManageModules = ({
     const includeFooter = true;
     return displayModal(null, body, includeFooter, footer);
   };
+  const textAndMultimediaIsChecked = enabledModules.some(
+    module => module.getIn(['moduleType', 'title']) === 'Text & Multi-media'
```

Comment:
You should use `module.getIn(['moduleType', 'identifier']) === 'INTRODUCTION'` title may change in the future.